uisdk/src/test/java/com/karhoo/uisdk/screen/booking/quotes/BookingQuotesPresenterTest.kt

```diff
@@ -24,6 +26,7 @@ class BookingQuotesPresenterTest {
     private val presenter: BookingQuotesMVP.Presenter = BookingQuotesPresenter(view)
     private val testContext: Context = mock()
     private var resources: Resources = mock()
+    private var isPrebook: Boolean = TripInfo().dateScheduled != null
```

Comment:
I think it would always be false as `TripInfo().dateScheduled` is always null 

to test different scenarios you need to assign value to `dateScheduled` so you can verify the prebook logic as well